common/edl/time.edl

```diff
@@ -14,8 +14,14 @@
 
 enclave
 {
-    include "openenclave/corelibc/time.h"
-    include "openenclave/internal/syscall/types.h"
+    // Needed for integral type time_t
```

Comment:
`time_t` is [implementation defined](https://en.cppreference.com/w/c/chrono/time_t) and has different underlying types in Windows and Linux. 
Use of non-portable types in EDL would lead to incorrect behavior.

Reply:
This doesn't change what type is used. If there is a bug now there was a bug before.

The reason this works is that on both Windows and Linux for x64 arch, time_t is defined as a 64 bit type.

Reply:
@anakrish please open an issue and we can fix it in another Pr